starlette/responses.py

```diff
@@ -19,9 +19,9 @@
 try:
     import ujson as json
 
-    JSON_DUMPS_OPTIONS = {"ensure_ascii": False}
+    JSON_DUMPS_OPTIONS: typing.Dict[str, typing.Any] = {"ensure_ascii": False}
 except ImportError:  # pragma: nocover
-    import json
+    import json  # type: ignore
```

Comment:
If mypy is forcing us to add this kind of noise then I'm :-/

Reply:
Yes, me too. Seems that is the only solution. You can take a look [here](https://github.com/python/mypy/issues/1153)